serverless/event_sources/knative-event-sources.adoc

```diff
@@ -9,20 +9,20 @@ An _event source_ is an object that links an event producer with an event _sink_
 
 Currently, {ServerlessProductName} supports the following event source types:
 
-ApiServerSource:: Connects a sink to the Kubernetes API server.
-PingSource:: Periodically sends ping events with a constant payload. It can be used as a timer.
-SinkBinding:: Allows you to connect core Kubernetes resource objects such as a `Deployment`, `Job`, or `StatefulSet` with a sink.
-KafkaSource:: Connect a Kafka cluster to a sink as an event source.
+API server source:: Connects a sink to the Kubernetes API server.
+Ping source:: Periodically sends ping events with a constant payload. It can be used as a timer.
+Sink binding:: Allows you to connect core Kubernetes resource objects, such as `Deployment`, `Job`, or `StatefulSet` objects, with a sink.
+Kafka source:: Connect a Kafka cluster to a sink as an event source.
 
 :FeatureName: Apache Kafka on {ServerlessProductName}
 include::modules/technology-preview.adoc[leveloffset=+2]
 
 You can create and manage Knative event sources using the **Developer** perspective in the {product-title} web console, the `kn` CLI, or by applying YAML files.
 
-* Create an xref:../../serverless/event_sources/serverless-apiserversource.adoc#serverless-apiserversource[ApiServerSource].
-* Create an xref:../../serverless/event_sources/serverless-pingsource.adoc#serverless-pingsource[PingSource].
-* Create a xref:../../serverless/event_sources/serverless-sinkbinding.adoc#serverless-sinkbinding[SinkBinding].
-* Create a xref:../../serverless/event_sources/serverless-kafka-source.adoc#serverless-kafka-source[KafkaSource].
+* Create an xref:../../serverless/event_sources/serverless-apiserversource.adoc#serverless-apiserversource[API server source].
+* Create an xref:../../serverless/event_sources/serverless-pingsource.adoc#serverless-pingsource[ping source].
+* Create a xref:../../serverless/event_sources/serverless-sinkbinding.adoc#serverless-sinkbinding[sink binding].
+* Create a xref:../../serverless/event_sources/serverless-kafka-source.adoc#serverless-kafka-source[Kafka source].
```

Comment:
ping, sink are low, but why is Kafka with capital K ?

Reply:
Because it's a proper name, e.g. Apache Kafka, as opposed to a generic object?

Reply:
ok.

speaking of "Apache KAfka" - that is the official trademark term. I think the first occurence on a page should have that /cc @swoodman  - instead of just Kafka

Reply:
Makes sense, updated it here and for `serverless-listing-event-sources.adoc`. Let's call this out as a follow up in any of Max's Kafka PRs too